pkg/controller/replication/replication_controller.go

```diff
@@ -478,10 +573,19 @@ func (rm *ReplicationManager) manageReplicas(filteredPods []*api.Pod, rc *api.Re
 		var wg sync.WaitGroup
 		wg.Add(diff)
 		glog.V(2).Infof("Too few %q/%q replicas, need %d, creating %d", rc.Namespace, rc.Name, rc.Spec.Replicas, diff)
+		// needs to set the gvk to generate ownerReference
```

Comment:
Acronym for "group version kind". It's used in many places in the code base. But if even you don't know it, I think it worth expanding.